core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -674,6 +677,17 @@ private[spark] class Executor(
     }
   }
 
+  private def setMDCForTask(taskDescription: TaskDescription): Unit = {
+    val properties = taskDescription.properties
+
+    org.slf4j.MDC.put("taskName", taskDescription.name)
```

Comment:
nit: you should import `org.slf4j.MDC` in the header of the file.

Reply:
Done